pkg/scheduler/scheduler.go

```diff
@@ -155,28 +159,28 @@ func (pc *Scheduler) loadSchedulerConf() {
 }
 
 func (pc *Scheduler) watchSchedulerConf(stopCh <-chan struct{}) {
-	if pc.fileWatcher == nil {
+	informerFactory := informers.NewSharedInformerFactory(pc.cache.Client(), 0)
+	cmInformer := informerFactory.Core().V1().ConfigMaps()
+	cmInformer.Informer().AddEventHandler(
+		cache.ResourceEventHandlerFuncs{
+			UpdateFunc: pc.updateConfigMap,
+		})
+	informerFactory.Start(stopCh)
+}
+
+// updateConfigMap update scheduler config
+func (pc *Scheduler) updateConfigMap(_, newObj interface{}) {
```

Comment:
is it possible to emit an error message for the user if they try to use a configmap but not specify a right one?